src/relion/_parser/autopick.py

```diff
@@ -10,13 +11,31 @@
     [
         "number_of_particles",
         "micrograph_full_path",
+        "mc_image_full_path",
         "first_micrograph_name",
+        "highlighted_micrograph",
+        "coordinates",
         "job",
     ],
 )
 
+ParticleCacheRecord = namedtuple(
+    "ParticleCacheRecord",
+    [
+        "data",
+        "file_size",
+    ],
+)
+
 
 class AutoPick(JobType):
+    def __init__(self, path, particle_cache=None):
+        super().__init__(path)
+        if particle_cache is None:
+            self._particle_cache = {}
+        else:
+            self._particle_cache = particle_cache
```

Comment:
If you're happy with excluding all falsey values (including empty dictionary references) then you can write
```suggestion
        self._particle_cache = particle_cache or {}
```